dts/bindings/sensor/ti,hdc2xxx.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2021, Aurelien Jarno
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments HDC2XXX Temperature and Humidity Sensor
+
+compatible: "ti,hdc2xxx"
+
+include: i2c-device.yaml
```

Comment:
It would probably make sense to include an optional field for the interrupts line.

E.g. https://www.ti.com/lit/ug/swau127a/swau127a.pdf

Reply:
Yes, that is planned, but at this stage the driver doesn't support it, so I don't think we should add it to the DT bindings. The plan is first to add basic support for the HDC2080, and then add advanced features.